diff --git a/meson.build b/meson.build
index 8b1d886..a8bc428 100644
--- a/meson.build
+++ b/meson.build
@@ -98,7 +98,7 @@ endif
 gtest_dep = dependency('gtest', main : true, version : '>=1.10', required : get_option('enable_testing'))
 gtest_main_dep = dependency('gtest_main', version : '>=1.10', required : get_option('enable_testing'))
 CLI11_dep = dependency('CLI11', fallback : [ 'cli11' , 'CLI11_dep' ])
-fmt_dep = dependency('fmt', version:'>=8.1.1')
+fmt_dep = dependency('fmt', version:'>=8.1.1', static: true)
 
 cmake = import('cmake')
 
